lib/raven/configuration.rb

```diff
@@ -261,6 +262,13 @@ def verify!
       end
     end
 
+    def verify_messages
```

Comment:
Exactly - and that's the issue, imo: we don't get any useful information, cause it will always be  `true`, i.e. we will never report a blocked capture because of the callback when testing with `nil`.